.eslintrc.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
   "rules": {
     "func-names": "off",
     "global-require": "off", // Interfers with optional and eventual circular references
+    "arrow-body-style": "off",
```

Comment:
Let's maybe maintain an alphabetical order (it's easier then to get around with growing list of rules)